### ISSUE TITLE
[Broken Website] Replace with title

### ISSUE THREAD

**Issue by stoyle** [darkreader/darkreader#10172]
# Broken Website Report

## Website Address
https://www.finn.no/realestate/homes/ad.html?finnkode=276041549

FINN is modernizing their ad pages. And it seems all new ad pages are boken (e.g. all classified ads accessed from https://www.finn.no/realestate/homes/search.html?is_new_property=false&sort=PUBLISHED_DESC)

## Steps To Reproduce
- Try to click "Lett til favoritt" -> nothing happens.
- Try to click right-left buttons in image gallery

## Problematic Observation
It seems the plugin interferes with javascript loading/initialization. Basically javascript does not run at all. The page is server side rendered, so content is there, but interactive elements do not work.

## System Information
- OS: macOS
- Browser: Chrome 106, Firefox 106
- Dark Reader Version: 4.9.58

**Comment by Gusted** [comment]
Hi @stoyle,

I'm unfortunately not able to reproduce this issue, is this issue persistence and reproducible on your part?

**Comment by stoyle** [comment]
Yeah it's completely persistent and reproducible. The object in the ad has been sold, so "legg til favoritt", is actually not clickable by default any more. Anyways, I hope the linked video shows the problem. Basically, for some reason, no javascript runs. Turning off dark reader, and it runs.

Video was to large 69MB, so couldn't embed it. Should be able to view it here though: https://mega.nz/file/YVFy3IRK#yMCMkl_Ep9LKjj4BVQeestBW-y3ZtYuy1EVwZmMNTQc

**Comment by Gusted** [comment]
Really really weird, it seems like Dark Reader is blocking other interactions from happening... Could you try to see if you can [make a performance profile](https://developer.chrome.com/docs/devtools/evaluate-performance/)? That should show what Dark Reader is doing.

**Comment by clandestinecephalopod** [comment]
Instead of creating a duplicate report, I am confirming that I also have the same issues as @stoyle. They are caused by darkreader conflicting with javascript. Disabling js with darkreader off has the same result as js on with darkreader active.

Example listing: https://www.finn.no/bap/webstore/ad.html?finnkode=293535750

**Problems**

1. Clicking the product image is supposed to bring it into a full-screen gallery mode. Instead, it loads a new page with all images in a list.
2. Left/right arrows on-screen are not clickable.
3. Left/right arrows on keyboard are supposed to scroll through the gallery, but only work if you first click just above/below the image. The image number (1/6) does not change either.
4. Buttons "Legg til favoritt" and "Del" or not clickable.
5. The text showing the number of people who have saved the listing is missing ("xx har lagt til som favoritt")

**SysInfo**

- Windows 11
- Firefox 110.0.1
- Darkreader 4.9.62

**Comment by danielmmmm** [comment]
I experience the same problem on Finn.no (a kind of Norwegian ebay) with Firefox 113.0.2 (64-bit) and Chrome 113.0.5672.127 (Official Build) (32-bit) on Windows 10.

With Dark Reader enabled, clickable elements simply do not work. The corresponding `<div>` or `<button>` HTML elements are missing their click event handler. (Here is an example product page with an  [advertisement for used ski boots](https://www.finn.no/bap/forsale/ad.html?finnkode=294623245).)
Interestingly, this affects only the content that is loaded initially, for the original product advertisement. Once the site has finished loading the code for the original advertisement, it will fetch additional code to show similar advertisements under "Andre ting på Torget". These additional advertisements have the same heart icon to add them to favourites as the original advertisement. But their heart icon actually works because the corresponding `<button>` has a click event handler.

All javascripts seem to get downloaded successfully, even with Dark Reader enabled. E.g. the arrows to view the next/previous product image are handled by a script in `assets.finn.no/pkg/recommerce-item-page/1.0.56/public/build/_shared/`, which is present.
Therefore, I guess it is an initialisation error more than a loading error.

I also noticed that the site is working fine when Dark Reader is disabled at the time the site gets requested from the server, but activated approximately 2 seconds later. When I activate Dark Reader after only 1 second of the page loading, clickable elements are not working, just like when the site is loaded with Dark Reader enabled from the very beginning.
I didn't see anything suspicious in the Firefox console. There were no differences in the console output of loading the site with versus without Dark Reader. I compared only the first 3 seconds of the page loading since this is the time frame within which something goes wrong.

(And, I disabled uBlock Origin and Violentmonkey, but that didn't make a difference.)
